backend/core/src/core/category/application/dtos/cast-to-search-response-dto.ts

```diff
@@ -0,0 +1,22 @@
+import { Video } from '@prisma/client';
+import { SearchByCategoryResponseDto } from 'shared/build';
+
+export const castToSearchByCategoryResponseDto = ({
+  id,
+  name,
+  description,
+  status,
+  videoPath,
+  liveViews,
+  videoViews,
```

Comment:
the search results should match BaseVideoResponseDto, check getAll() in video repository adapter to see how the data should be retrieved